cmd/create/operatorroles/by_prefix.go

```diff
@@ -398,6 +406,16 @@ func createRolesByPrefix(r *rosa.Runtime, prefix string, permissionsBoundary str
 		if err != nil {
 			return err
 		}
+
+		if len(policyArns) > 0 {
```

Comment:
Is it required to check the length? If there are none the loop will just be skipped anyway

Reply:
+1, the check is unecessary